### DISCUSSION TITLE
Copilot is not available in your location

### DISCUSSION THREAD

**Discussion by niccoloobassan** [community/community#51881] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Good evening everyone,
I live in Italy and I have this problem using GitHub Copilot on VS Code: GitHub Copilot could not connect to server. Extension activation failed: "At this time, Copilot is not available in your location"

[INFO] [auth] [2023-04-03T19:25:12.915Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-04-03T19:25:12.916Z] GitHub Copilot could not connect to server. Extension activation failed: "At this time, Copilot is not available in your location"

Do you know how I can fix?
Thank you in advance.

**Comment by Akash1134** [comment]
Hi @niccoloobassan ,

Thanks for getting in touch.

Due to current trade controls, your GitHub account has been restricted from making use of Copilot. This is a region-based restriction and applies to all users located in some trade-restricted countries, being flagged for the same earlier, and is separate from other GitHub services. You can continue to use other GitHub services including your Pro plan. You can read more about it here:

[GitHub and Trade Controls](https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls)

Sorry if that wasn't what you were hoping to hear.

If you feel this should not be the case, feel free to contact our Support team and [open a ticket ](https://support.github.com/request?tags=dotcom-footer)with them. Thanks!

**Reply by sy-python** [reply]
Nothing in the link even mentions Copilot.

**Reply by Akash1134** [reply]
Hi @SansCipher The GitHub Trade Restrictions are applied on all GitHub services and include Copilot as well.

**Reply by sy-python** [reply]
And yet I can still post code on github and stuff. Besides, I wasn't even paying for Copilot, it was a part of education package. But whatever, yall clearly don't care about Russian developers.

**Comment by Akash1134** [comment, marked as answer]
Hi @niccoloobassan and @SansCipher , Could you please logout of Copilot from your IDE and log back in to see if that helps? Thanks!

**Reply by riZZZhik** [reply]
Hello
As expected, it didn't help

**Reply by sy-python** [reply]
Huh. Surprisingly, it worked

**Reply by riZZZhik** [reply]
@SansCipher Where are you from and can you check for vpn please?

**Reply by p-yavari** [reply]
Hello @Akash1134 ,
That works for me 😃👌. 
Thanks.

**Comment by mvahedii** [comment]
I also have this problem
But why should developers be sanctioned?

**Reply by waelnassaf** [reply]
Because as a student making a living freelancing, I pose a great threat to US national security, I could just be developing a nuclear program here in my basement
The whole sanctions idea is just stupid, I hope they'll be lifted on Syria soon
